cypress/e2e/tests/migration/task-manager/sorting.filter.pagination.test.ts

```diff
@@ -139,10 +146,43 @@ describe(["@tier3"], "Filtering, sorting and pagination in Task Manager Page", f
     });
 
     it("Sorting tasks", function () {
+        // Ensure total pod count does not exceed the number of tackle pods.
+        // Existing pods are not automatically deleted.
+        deleteApplicationTableRows();
+        getNumberOfNonTaskPods().then((podsNum) => {
+            limitPodsByQuota(podsNum);
+        });
+        let bookServerApp: Analysis;
+        cy.fixture("application").then((appData) => {
+            cy.fixture("analysis").then((analysisData) => {
```

Comment:
These fixtures are already loaded at this point because you moved them to the beforeEach hook

Reply:
Thanks! I will fix that